nvflare/private/fed/server/fed_server.py

```diff
@@ -729,9 +729,7 @@ def _check_server_state(self, overseer_agent):
             self._turn_to_hot()
 
         elif isinstance(self.server_state, Hot2ColdState):
-            self._turn_to_cold()
-
-        self._notify_state_change(old_state_name)
```

Comment:
Why we don't need to notify all the other state changes?

Reply:
Only when the server is Hot state it has the running job process, and notify to turn to Cold.

Reply:
got it, then we could rename it to `_notify_state_change_to_cold` or something, but that can be done in another PR